html/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include avocado_result_html/resources *
```

Comment:
One more thing, I don't think we should put those real plugins directly into the main directory. How about creating `optional_plugins` directory and use `Makefile` to walk it and deploy them based on a variable?

```
make OPTIONAL_PLUGINS=html deploy
```
would install the html